_posts/patterns/2015-04-09-buttons.md

```diff
@@ -19,78 +19,85 @@ info: Buttons should be used for performing actions within the edX environment.
 </div>
 
 <h3 class="hd-6 example-set-hd">Elevated (Alt)</h3>
 <div class="example-set">
-    <button type="button" class="btn btn-elevated-alt btn-base">Elevated (Alt) Base</button>
-    <button type="button" class="btn btn-elevated-alt btn-large">Elevated (Alt) Large</button>
-    <button type="button" class="btn btn-elevated-alt btn-small">Elevated (Alt) Small</button>
-    <button type="button" class="btn btn-elevated-alt btn-base" disabled>Elevated (Alt) Disabled</button>
+    <button type="button" class="btn-elevated-alt btn-base">Elevated (Alt) Base</button>
+    <button type="button" class="btn-elevated-alt btn-large">Elevated (Alt) Large</button>
+    <button type="button" class="btn-elevated-alt btn-small">Elevated (Alt) Small</button>
+    <button type="button" class="btn-elevated-alt btn-base" disabled>Elevated (Alt) Disabled</button>
 </div>
 
 <h3 class="hd-6 example-set-hd">Combo</h3>
 <div class="example-set">
-    <button type="button" class="btn btn-primary btn-base btn-combo">Combo Base</button>
-    <button type="button" class="btn btn-primary btn-base btn-combo">Combo Base</button>
-    <button type="button" class="btn btn-primary btn-base btn-combo">Combo Base</button>
+    <button type="button" class="btn-primary btn-base btn-combo">Combo Base</button>
+    <button type="button" class="btn-primary btn-base btn-combo">Combo Base</button>
+    <button type="button" class="btn-primary btn-base btn-combo">Combo Base</button>
 </div>
 
 <!--<h3 class="hd-6 example-set-hd">With Icons</h3>
 <div class="example-set">
-    <button type="button" class="btn btn-default btn-base has-icon">Default Base with Icon</button>
-    <button type="button" class="btn btn-default btn-base has-icon icon-reversed">Default Base with Icon Reversed</button>
+    <button type="button" class="btn-default btn-base has-icon">Default Base with Icon</button>
+    <button type="button" class="btn-default btn-base has-icon icon-reversed">Default Base with Icon Reversed</button>
 </div>-->
 
 <h3 class="hd-6 example-set-hd">Overlay</h3>
 <div class="example-set">
     <div class="button-overlay-demo">
-        <button type="button" class="btn btn-overlay btn-base">Overlay Base</button>
-        <button type="button" class="btn btn-overlay btn-large">Overlay Large</button>
-        <button type="button" class="btn btn-overlay btn-small">Overlay Small</button>
-        <button type="button" class="btn btn-overlay btn-base" disabled>Overlay Disabled</button>
+        <button type="button" class="btn-overlay btn-base">Overlay Base</button>
+        <button type="button" class="btn-overlay btn-large">Overlay Large</button>
+        <button type="button" class="btn-overlay btn-small">Overlay Small</button>
+        <button type="button" class="btn-overlay btn-base" disabled>Overlay Disabled</button>
     </div>
 </div>
 
 <h3 class="hd-6 example-set-hd">Links</h3>
 <div class="example-set">
+<<<<<<< HEAD
```

Comment:
![giphy](https://cloud.githubusercontent.com/assets/163763/10693738/248c1d64-7968-11e5-9c30-b54396c9e388.gif)

**D'oh!** Thanks for the catch!